<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayProductWindow</class>
 <widget class="QDialog" name="DisplayProductWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="goBack">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>610</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1131</width>
     <height>592</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ram</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetRam">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type of memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ram Timing</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tension</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cpu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetCpu">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type of processor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Socked</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number of cors</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Core timing</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Motherboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetMotherboard">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>147</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cpu Famili Support</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chipset</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Procesor Socked</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
